### ISSUE TITLE
Mozilla Privacy Arcade

### ISSUE THREAD

**Issue by chadsansing** [mozilla/global-sprint#33]
**[ Project Contact ]** @chadsansing
**[ GitHub Repo ]** https://github.com/MozillaFoundation/mozilla-privacy-arcade
**[ Track ]** Privacy + Security, Digital Inclusion
**[ Community ]** Internet of Things - wrangler: @thornet
**[ Location ]** Virginia, USA, Eastern Time Zone
### Description
Help people learn about online safety and inclusion in fun, low-risk ways!
***

## Want to contribute to this project during #mozsprint?
Join us at the [Global Sprint, June 1-2](https://mozilla.github.io/global-sprint). Leave a comment below if you're interested in contributing to this project during #mozsprint 2017!

***

## Note to the Project Lead :tada:
Congrats, @chadsansing! This is your official project listing for the [Mozilla Global Sprint 2017](https://mozilla.github.io/global-sprint). To confirm your registration, please complete and check off the following:

- [x] Complete [Open Leadership 101](https://mozilla.teachable.com/p/open-leadership-101)
- [x] Provide a GitHub repository for work and discussion on your project in a comment
  * new to GitHub?  Here's a [step-by-step guide on using the #mozsprint template](https://mozilla.github.io/global-sprint/project-lead/templates/)
- [x] Create a README file in your project repository. This file should help newcomers understand what your project is, why it's important, and kinds of help you're looking for.
* [x] [Create file: LICENSE](http://choosealicense.com/) to give your project an open license, allowing for sharing and remixing.
- [x] Turn on your [Issue Tracker](https://mozilla.github.io/global-sprint/project-lead/templates/#4-create-issues) and create issues to describe each task that you need help with and how a contributor can get started on that task. [Create a label](https://mozilla.github.io/global-sprint/project-lead/templates/#5-add-a-mozsprint-label) called `mozsprint` and apply it to your issues.

#### Checklist for FEATURED Projects :clipboard:
To have your project FEATURED on [Mozilla Pulse](http://mozillapulse.org/), complete the following documentation. In past Sprints, well-documented featured projects have 5 times more contributions than other projects. Details about each item and more information about how to create them are [on our Project Requirements Page](https://mozilla.github.io/global-sprint/project-lead/project-requirements/). 

* [x] In your README, link to the [Mozilla Community Participation Guidelines](https://www.mozilla.org/en-US/about/governance/policies/participation/) or write your own code of conduct.
* [x] Create file: CONTRIBUTING.md so others know how they can contribute. If you'd like, you can [remix this template](https://github.com/acabunoc/mozsprint-repo-template/blob/master/CONTRIBUTING.md)
* [x] Comment with an image to use on Mozilla Pulse for this project listing

Once all of the above is complete,
- [x] Leave a comment with the text `This is ready for Mozilla Pulse`. Your community wrangler will review this issue and post your project :balloon:

If you get stuck at any point, feel free to look at the [requirements page](https://mozilla.github.io/global-sprint/project-lead/project-requirements/) and [project templates](https://mozilla.github.io/global-sprint/project-lead/templates/) or reach out to your community wrangler, @thornet. We're here to help you through this process.

**Comment by chadsansing** [comment]
![mozilla-privacy-arcade-1](https://cloud.githubusercontent.com/assets/4128600/25043317/f78751b6-20ec-11e7-92e0-a848831b8bee.png)

**Comment by chadsansing** [comment]
This is ready for Mozilla Pulse.

**Comment by arkodyuti** [comment]
Hello. Looking forward to work on this project.

**Comment by chadsansing** [comment]
Awesome, @arkodyuti - be sure to visit each challenge link [here](https://github.com/MozillaFoundation/mozilla-privacy-arcade) (Cryptomancer, Game Jam, IoT Escape Room, and Offline Games) and to comment in the repo of the project that interests you the most.

**Comment by arkodyuti** [comment]
Thanks for the links Chad. Will look into this and get back to you soon.

-Cheers
Arko

**Comment by chadsansing** [comment]
@thornet, would you Pulse this, or would you rather I did? Not sure of #mozsprint etiquette here as you are my official community/project wrangler!

**Comment by chadsansing** [comment]
NVM, @thornet! I'm going to split this up into 4 #mozsprint issues/repos and pulse them myself at @zee-moz's suggestion.

**Comment by abbycabs** [comment]
Thanks for participating in #mozsprint 2017! We're gearing up for 2018 and closing all the issues from last year. Want to bring your project to #mozsprint 2018, May 10-11? [Our Call for Projects](https://foundation.mozilla.org/opportunity/global-sprint/for-project-leads/) is open!
